lib/puma/runner.rb

```diff
@@ -85,10 +85,11 @@ def output_header(mode)
       min_t = @options[:min_threads]
       max_t = @options[:max_threads]
 
-      log "Puma starting in #{mode} mode..."
-      log "* Version #{Puma::Const::PUMA_VERSION} (#{ruby_engine}), codename: #{Puma::Const::CODE_NAME}"
-      log "* Min threads: #{min_t}, max threads: #{max_t}"
-      log "* Environment: #{ENV['RACK_ENV']}"
+      log "Puma starting in #{mode} mode...\n" \
+          "* Version #{Puma::Const::PUMA_VERSION} (#{ruby_engine}), codename: #{Puma::Const::CODE_NAME}\n" \
+          "* Min threads: #{min_t}, max threads: #{max_t}\n" \
+          "* Environment: #{ENV['RACK_ENV']}\n" \
+          "*         Pid: #{Process.pid}"
```

Comment:
I like this addition 👍

```suggestion
          "*         PID: #{Process.pid}"
```

How it would look like live:

```
$ bundle e bin/puma -p0 -C test/config/app.rb
Puma starting in single mode...
* Version 5.1.0 (ruby 2.7.1-p83), codename: At Your Service
* Min threads: 0, max threads: 5
* Environment: development
*         PID: 69955
* Listening on http://0.0.0.0:50084
Use Ctrl-C to stop
```


Some alternatives worth considering?

```
$ bundle e bin/puma -p0 -C test/config/app.rb
Puma starting in single mode...
* Version 5.1.0 (ruby 2.7.1-p83), codename: At Your Service
* Min threads: 0, max threads: 5
* Environment: development
* Process ID:  75834
* Listening on http://0.0.0.0:50746
Use Ctrl-C to stop
```

```
$ bundle e bin/puma -p0 -C test/config/app.rb
Puma starting in single mode...
* Version 5.1.0 (ruby 2.7.1-p83), codename: At Your Service
* Min threads: 0, max threads: 5
* Environment: development
* PID:         76584
* Listening on http://0.0.0.0:50824
Use Ctrl-C to stop
```

Reply:
Not a big issue, I mostly wanted it for development & CI.  Also, we're supposed to know Puma, so maybe:
```
* Environment: development
* Main PID:    76584
- or -
* Environment: development
*    Main PID: 76584
```
Thought of adding 'Main' simple because more pids exist when in cluster mode?

Reply:
I like it 👍🏻

Reply:
Anyone have a preference for one or the other (of the two above with `Main PID:`?

Reply:
For me, "Main" is confusing because I don't know what it refers to in the context of Puma.

This is the PID of the master process right? So "Master PID" makes sense.

Reply:
Oh actually, a one line log here looks bad in cluster mode.

```
[58881] Puma starting in cluster mode...
* Version 5.1.0 (ruby 2.7.2-p137), codename: At Your Service
* Min threads: 0, max threads: 5
* Environment: development
[58881] *  Master PID: 58881
[58881] * Process workers: 2
```

Reply:
Master is ok.  Prefer first or second?  I used main only because 'master'...
```
* Environment: development
* Master PID:  76584
- or -
* Environment: development
*  Master PID: 76584
```

Reply:
> Oh actually, a one line log here looks bad in cluster mode

Maybe you added the string differently?  I don't think the code here can output `[58881] *  Master PID: 58881`?

It is repetitive in cluster mode